components/create-collective/index.js

```diff
@@ -195,7 +195,7 @@ const createCollectiveMutation = gqlV2/* GraphQL */ `
   }
 `;
 
-const addCreateCollectiveMutation = graphql(createCollectiveMutation, {
+export const addCreateCollectiveMutation = graphql(createCollectiveMutation, {
```

Comment:
Good work exporting and reusing the existing mutations instead of creating new ones 👍